subsys/bluetooth/shell/iso.c

```diff
@@ -417,7 +465,7 @@ static int cmd_big_term(const struct shell *shell, size_t argc, char *argv[])
 #endif /* CONFIG_BT_ISO_BROADCAST */
 
 SHELL_STATIC_SUBCMD_SET_CREATE(iso_cmds,
-	SHELL_CMD_ARG(bind, NULL, "[dir] [interval] [packing] [framing] "
+	SHELL_CMD_ARG(bind, NULL, "[dir=<,>,<>] [interval] [packing] [framing] "
```

Comment:
I'm  a bit worried whether this is something people will understand. Is the `>`, `<` and `<>` terminology used other places that e.g. BTMON?

Reply:
We can either use that or tx,rx,txrx.

Reply:
I would go for tx, rx, txrx. I have never understood how one knows what < and > means :-)